### PR TITLE
Cssr table changes

### PR DESCRIPTION
**Pull request by Shakir-Sopra**
@wozzer72 

Hi Warren

I have done all the changes what you advised in your last comments and updated the ddl for CssR table.
CSSRID has been made primary key.
Delete command updated.

I am working on trigger NMDSID trigger, will come seperately.

Thanks
Shakir

### CODE REVIEW

File: DEV/dev.db.sql [hunk 1]
```diff
@@ -1 +0,0 @@
-\i ../create_db_ddl.sql
```
Comment (wozzer72) [inline DEV/dev.db.sql]: Why have you deleted this?

File: TST/tst.db.sql [hunk 2]
```diff
@@ -1 +0,0 @@
-\i ../create_db_ddl.sql
```
Comment (wozzer72) [inline TST/tst.db.sql]: Again. Why deleted?

File: UAT/uat.db.sql [hunk 3]
```diff
@@ -1 +0,0 @@
-\i ../create_db_ddl.sql
```
Comment (wozzer72) [inline UAT/uat.db.sql]: Again. Why deleted?

File: DEV/dev.db.sql (hunk 1 above)
Comment (Shakir-Sopra) [reply DEV/dev.db.sql]: That must be an accident as I have moved cleared my client branch if you found any folder deleted.
Please do not merge I will correct my branch and then you can merge it. However please advise as I have worked on create and delete script only.

Thanks
~Shakir

**Comment by Shakir-Sopra** [comment]
> The delete DDL is good. The revised primary key is good.
> 
> But where is the actual CSSR table; expecting to see lots of insert statements like `Services` and `LocalAuthoirty` - lines 670 to 997.

**Comment by Shakir-Sopra** [comment]
Hi Warren thanks for comments.

I have to create the insert statement and trigger nmdsid though.

For CSSR column, I might have to get data from Oracle converted cssr table.

I will work out the insert statement and let you know.

Thanks
Shakir

**Comment by wozzer72** [comment]
Updated PR, with the revised CSSR table, now replacing LocalAuthorities, which impacts on `EstablshmentLocalAuthority`.

**Comment by wozzer72** [comment]
Updated PR, with the revised CSSR table, now replacing LocalAuthorities, which impacts on `EstablshmentLocalAuthority`.

**Comment by wozzer72** [comment]
[NMDS-ID - displayed on Establishments](https://trello.com/c/Ept21xD8/357-nmds-id-displayed-on-establishments)

**Comment by Shakir-Sopra** [comment]
Hi Warren

I have reviewed the all CSSRID related table creation and updates to the localAuthority table and implemented the same in DEV as guided by the release notes.

All is implemented and verified hence approving the branch changes.

Thanks
Shakir

**Pull request merged**